src/Classes/MyRadio/AuthUtils.php

```diff
@@ -340,6 +427,25 @@ public static function addActionPermission($module, $action, $permission)
         );
     }
 
+    /**
+     * Deletes action permissions.
+     *
+     * @param int $actPermissionID     The action permission ID.
+     */
+    public static function removeActionPermission($actPermissionID = -1)
+    {
+        if ($actPermissionID >= 0) {
```

Comment:
why bother with the check (and default parameter) ? Seems like it's just asking for exceptions